### PR TITLE
Send an empty location if location is disabled or location consent is not given

### PR DESCRIPTION
**Pull request by ijunaid**

### CODE REVIEW

**Review by arifBurakDemiray — COMMENTED** [review]
this looks ok for initilization

but what about location consent revoked after init?
It should also send that info

**Review by turtledreams — CHANGES_REQUESTED** [review]
changelog

File: CountlyConsentManager.m [hunk 1]
```diff
@@ -350,6 +350,8 @@ - (void)setConsentForLocation:(BOOL)consentForLocation
     else
     {
         CLY_LOG_D(@"Consent for Location is cancelled.");
+        
+        [CountlyConnectionManager.sharedInstance sendLocationInfo];
```
Comment (arifBurakDemiray) [inline CountlyConsentManager.m:354]: Do we guarantee that we are sending empty location info?

**Review by arifBurakDemiray — CHANGES_REQUESTED** [review]
Please also add some tests about all location situations.

**Pull request merged**